### ISSUE TITLE
refactor skipLink component

### ISSUE THREAD

**Issue by tobi-or-not-tobi** [SAP/cloud-commerce-spartacus-storefront#2730]
- remove hardcoded links in skipLink component
- introduce a skipLinkService that can be used by individual components to register a skipLink
- improve UI for the skipLinks

**Comment by tobi-or-not-tobi** [comment]
@Xymmer  Since this component is not ready, I recommend to remove it entirely for 1.0 and add it as a new feature in a 1.x release.

**Comment by tobi-or-not-tobi** [comment]
We've discussed this feature and had a number of thoughts:
- use a clear name, `skipLink` seems to be right
- we should not hardcode any skipLinks as we did previously. We like a configurable solution, that customers can reuse to skip other portions of the page
- we should not use anchor links, since these are driven by page IDs. We do not use page IDs in single page app, where components can be reused. We should use the [scrollTo API](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo).
- this functionality is about skipping portions of the page. This is not so much related to components, but more related to page sections. We should connect the skipLinks to page slots, or page groups (header, footer or pageTemplates). This is all related to the `PageLayoutService`. 
- we like @developpeurweb to provide the skipLinks we need for Spartacus as well as a few ideas we see at existing commerce sites. Based on this, we can confirm whether the solution we come up with is valid.
- would be great if @kacperknapik and @Zeyber can come up with a solution proposal. I'd like to be aligned with the solution.

**Comment by kacperknapik** [comment]
@tobi-or-not-tobi please check #5048 (PR #5128) and decide if this ticket can remain closed or should be reopened

**Comment by kacperknapik** [comment]
deprecated by https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5128#issuecomment-554029794
